float64 x_normal
float64 y_normal
float64 z_normal
float64 rotation_angle
---

float64[] quat 

